packages/expo/src/Facebook/ExponentFacebook.web.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
?

Reply:
Should we write something that injects this script, or would it be better to just hardcode it in the `index.html` also if you have no idea that's fine too